packages/bitstringStatusList/src/bitstring-status-list-status.ts

```diff
@@ -188,6 +189,7 @@ const verifyBitstringStatusListCredential = (
           new VeramoEd25519Signature2018(),
           new VeramoJsonWebSignature2020(),
           new VeramoEcdsaSecp256k1RecoverySignature2020(),
+          new VCkitEddsaRdfc2022(),
```

Comment:
Why do we need to add the VCkitEddsaRdfc2022 crypto suite to the bitstring status list?

Reply:
I added the `new VCkitEddsaRdfc2022()` line of code here because when the DID document of `did:web`
uses `Multikey`, it can use the `VCkitEddsaRdfc2022` class to get the corresponding suite (DataIntegrityProof) for verifying the VC.

Could you tell me your concerns about this implementation? Thanks.